contrib/run.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import asyncio
+import subprocess
+
+HOST_COLOR = "\033[94m"
+CLIENT_COLOR = "\033[93m"
+ENDC = "\033[0m"
+
+def log(who: str, color: str, line: str):
+    print(f"{color}{who.ljust(8)}{ENDC} {line}", end="")
+
+async def run_client(room_id: str):
+    cmd = ["./target/debug/netcanv", "join-room", "-r", room_id]
```

Comment:
This script executes the app directly rather than delegating to `cargo run`. I think it would be better if the script did that + forwarded all positional arguments passed to it into `cargo` (assuming we use `argparse` for argument parsing within the script).

The biggest advantage of this approach would be that `cargo run` builds the app for you, so if you've made any changes it'll automatically pick them up and rebuild the app before running the client+host configuration. The other upside would be that you can do `python contrib/run.py -- --release` to run a release build instead of debug.

Reply:
Why do you think that?

You'd have to increase the complexity of the script such that it detects a `--release` argument and passes it onto `cargo build`, and then chooses the appropriate folder to run the app from. Feels a bit unnecessary to me if `cargo run` can do it.

Reply:
Sure, it's a contrib script anyways so I'm not gonna be that dictatorial about it.